update(content): add instructions for installing rules and falcoctl mentions (#1100)
falcosecurity/falco-website: content/en/docs/rules/default-custom.md

weight: 20
---

Falco comes with a default yaml rules file that is loaded if no specific configuration is provided, but that can be completely customized in several ways, depending on how Falco is installed. There are several ways in which you can specify the location of your custom rules and also download them and keep them up to date.

## The configuration file

The default configuration file, [`/etc/falco/falco.yaml`](https://github.com/falcosecurity/falco/blob/master/falco.yaml) makes Falco load rules from the `/etc/falco/falco_rules.yaml` file followed by any custom rules located in the `/etc/falco/rules.d` directory. This configuration is governed by the `rules_file` key:

```
rules_file:
  - /etc/falco/falco_rules.yaml
  - /etc/falco/falco_rules.local.yaml
  - /etc/falco/rules.d
```

Changing these configuration entries will affect the location and the order of the files that are downloaded.

## The command line

If you are running Falco directly from the command line, you can use the `-r` switch to add as many rules files as needed. e.g.:

```
# falco -r /path/to/my/rules1.yaml -r /path/to/my/rules2.yaml
```

## Falcoctl

> The [falcoctl](https://github.com/falcosecurity/falcoctl) tool provides functionality to download and update rules files when distributed as OCI artifacts to directories that can be specified by the user. The `install` command of the [falcoctl](https://github.com/falcosecurity/falcoctl) tool will download rulesfile OCI packages to a configurable directory (by default, that is `/etc/falco`). For instance, in order to install a specific version of the default rules file in `/etc/falco` you can run

Reviewer: ```suggestion
The [falcoctl](https://github.com/falcosecurity/falcoctl) tool provides functionality to download and update rules files distributed as OCI artifacts. The `install` command of the [falcoctl](https://github.com/falcosecurity/falcoctl) tool will download rules files to a configurable directory (by default, that is `/etc/falco`). For instance, to install a specific version of the default rules file in `/etc/falco` you can run the following commands:
```
